src/components/content/Notification.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="notification" :class="classType">
+    <button @click="triggerClose(notification)" class="delete" aria-label="Close notification" type="button"></button>
+    {{ notification.text }}
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Notification',
+  props: [
+    'notification',
+  ],
+  data() {
+    return {
+      timer: null,
+    };
+  },
+  created() {
+    const timeout = Object.prototype.hasOwnProperty.call(this.notification, 'timeout') ? this.notification.timeout : true;
+    if (timeout) {
+      const delay = this.notification.delay || 4000;
+      this.timer = setTimeout(() => {
+        this.triggerClose(this.notification);
```

Comment:
You already have a reference to `this.notification` on the component, pass it as a parameter to `triggerClose()` is redundant, I suggest this:

```javascript
this.timer = setTimeout(this.triggerClose);
```